tools/specs/Microsoft.Data.SqlClient.nuspec

```diff
@@ -46,7 +46,6 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.5.0" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="System.Configuration.ConfigurationManager" version="8.0.1" exclude="Compile" />
-        <dependency id="System.Text.Json" version="8.0.5" />
       </group>
       <group targetFramework="net6.0">
```

Comment:
@mus65 FYI, .NET 6 support was just dropped from the main branch today.

Reply:
I merged main and fixed the conflict.